compose-examples/prysm-only/create-account.yaml

```diff
@@ -3,14 +3,14 @@ services:
   validator-import-launchpad:
     container_name: validator-import-launchpad
     image: gcr.io/prysmaticlabs/prysm/validator:${PRYSM_DOCKER_TAG}
-    command: accounts import --keys-dir="/launchpad/eth2.0-deposit-cli/validator_keys" --wallet-dir="/data/wallets" --wallet-password-file="/data/passwords/wallet-password"
+    command: accounts import --keys-dir="/launchpad/eth2.0-deposit-cli/validator_keys" --wallet-dir="/data/wallets" --wallet-password-file="/data/passwords/wallet-password" --pyrmont
```

Comment:
thank you for your PR, since we use https://pyrmont.launchpad.ethereum.org/ to create validator keys, these are only valid for pyrmont network, also pyrmont network is configured on beacon node.

Reply:
ok, thanks. I wasn't sure if this was really needed.
When I went over the official [prysm documentation](https://docs.prylabs.network/docs/testnet/pyrmont#step-5-import-your-validator-accounts-into-prysm) they added the --pyrmont flag.